man/mustache.5.ron

```diff
@@ -146,32 +250,54 @@ Hash:
 
 Output:
 
-    <b>resque</b>
-    <b>hub</b>
-    <b>rip</b>
+      <b>resque</b>
+      <b>hub</b>
+      <b>rip</b>
+
+The same effect as above can be obtained without nested objects, by using
+the implicit iterator (see **Variables** above).
+
+Template:
+
+    {{#repo}}
+      <b>{{.}}</b>
+    {{/repo}}
+
+Hash:
+
+    {
+      "repo": ["resque", "hub", "rip"]
+    }
+
+Output:
+
+      <b>resque</b>
+      <b>hub</b>
+      <b>rip</b>
 
 **Lambdas**
```

Comment:
I think it would be helpful to move the Lambda's section to the bottom, along with other optional extensions (just inheritance, at the moment), in order to keep them self-contained.

Reply:
The manpage is currently organized by tag type rather than by spec feature. I'm not necessarily opposed to changing the structure, but I'd prefer to keep the current PR focused on just making the manpage up-to-date. Regardless, I think the current structure is user-friendly.